packages/lit-dev-content/samples/examples/react-basics/react.ts

```diff
@@ -0,0 +1,12 @@
+declare global {
+  interface Window {
+    React: any;
```

Comment:
Since these are just typed as `any` could we just do `export const React = (window as any).React;` below?

Alternatively, can we import and use the React types?

Reply:
i changed to the window cast like you said :)

As far as types, I tried importing from skypack unpackge and was unsuccessful. Will start a follow up PR to work on that.